### ISSUE TITLE
Add component select on a per component basis

### ISSUE THREAD

**Issue by maraisr** [brycehanscomb/semanticator#2]
Is there at all a future feature to filter based on a component? Check out [Semantic Commits](https://github.com/Russian60/git-semantic-commits) by @Russian60

**Comment by brycehanscomb** [comment]
Absolutely. When I get the initial flurry of building an MVP out, I'll be adding the following features:

* Search/filter by component (or lack of component)
* Copy-paste-able git commands to make an un-semantic commit into a semantic commit (probably with `rebase`)
* Pre-fill the repo `input` by passing it in by url (a la `?url=REPO`)
* Keeping a history of repos for quick loading & reference
* Stats / graphs about how semantic the repo is (and how much work is required to get to 100%)
* Maybe a [Git Badge](http://shields.io/) for people who care enough to show off their stats.

**Comment by maraisr** [comment]
Is this repo open for contributing? Wouldn't mind, forking and helping with features...

**Comment by brycehanscomb** [comment]
Just gave you push access. Feel free to work on branches and I'll merge things in whenever they're ready.

**Comment by maraisr** [comment]
I'll PR everything.
